app/js/arethusa.core/services/dependency_loader.js

```diff
@@ -1,35 +1,61 @@
 "use strict";
 
+/**
+ * The service is a wrapper for $ocLazyLoad that can be applied to arrays and makes sure paths are URLs
+ */
 angular.module('arethusa.core').service('dependencyLoader', [
   '$ocLazyLoad',
   '$q',
   'BASE_PATH',
   function($ocLazyLoad, $q, BASE_PATH) {
-    function expand(p) {
-      if (aU.isUrl(p)) {
-        return p;
-      } else {
-        return BASE_PATH + '/' + p;
-      }
-    }
-    function expandPath(path) {
-      var res = [];
-      if (angular.isArray(path)) {
-        return aU.map(path, expand);
-      } else {
-        if (angular.isString(path)) {
-          return expand(path);
+
+    /**
+     * Use $ocLazyLoad after assuring paths are URLs
+     * @param args
+     * @returns {*}
+       */
+    this.load = function(args) {
+      return $ocLazyLoad.load(expandPath(args));
+
+      /**
+       * Apply URL conversion to different kinds of path containers
+       * @param path
+       * @returns {*}
+       */
+      function expandPath(path) {
```

Comment:
@fbaumgardt expandPath is also called by loadInOrder ... needs to be moved back out into a class method?